server/routes/user/handlers.js

```diff
@@ -45,6 +45,7 @@ exports.create = {
         id: Joi.string().description('id of the earned achievement'),
         date: Joi.date().description('date of its receipt')
       })),
+      type: Joi.string().description('Type of the user, can be: attendee, company, or organization'),
```

Comment:
It's not very relevant, but this values should not be hardcoded, and instead fetched from the config file.